service/history/transferQueueStandbyTaskExecutor.go

```diff
@@ -144,7 +144,7 @@ func (t *transferQueueStandbyTaskExecutor) processActivityTask(
 		}
 
 		if activityInfo.StartedId == common.EmptyEventID {
-			return newPushActivityToMatchingInfo(*activityInfo.ScheduleToStartTimeout), nil
+			return newPushActivityToMatchingInfo(activityInfo.TaskQueue, *activityInfo.ScheduleToStartTimeout), nil
```

Comment:
so `activityInfo.TaskQueue` always equals to `task.TaskQueue?` Do you happen to know why we persist the field in transfer tasks in the first place?

Reply:
Updated to task.TaskQueue